### ISSUE TITLE
Hard telescope dependency breaks this plugin when using fzf-lua.

### ISSUE THREAD

**Issue by cjnucette** [jalvesaq/dict.nvim#1]
I guess there should be an option to select a different picker.

**Comment by jalvesaq** [comment]
I agree! Can you make a pull request for this? Telescope works fine for me and I don't have spare time to develop an alternative.
